js/templates/miniProfile.html

```diff
@@ -1,9 +1,12 @@
 <div class="flexVCent">
   <div>
-    <div class="userIcon disc clrBr2 clrSh1" style="<%= ob.getAvatarBgImage(ob.avatarHashes || {}) %>"></div>
+    <% const userLink = ob.peerIDx ? `href="#${ob.peerID}"` : ''; %>
```

Comment:
No, that was debugging code to see what happened when it failed, I forgot to take it out.